corehq/apps/export/tests/test_export_data_schema.py

```diff
@@ -1213,7 +1213,7 @@ def _create_schema(self, items):
         )
 
     def _assert_item_order(self, schema, path, items):
-        group_schema = list(filter(lambda gs: gs.path == path, schema.group_schemas))[0]
+        group_schema = list([gs for gs in schema.group_schemas if gs.path == path])[0]
```

Comment:
this is already a list? I don't think you need the explicit `list`

Reply:
https://github.com/dimagi/commcare-hq/pull/21802/commits/648bb354f84a1591709455720129f28ab20b3359